packages/app/src/app/overmind/namespaces/checkout/constants.ts

```diff
@@ -9,7 +9,6 @@ export const FREE_PLAN: PricingPlan = {
   price: 0,
   credits: 400,
   creditsNote: 'Ideal for hobbyists using Devboxes up to 40 hours a month.',
-  sandboxes: 20,
```

Comment:
we still have a limit for free sandboxes

Reply:
yes, but this is not needed in this data type, as it is not the one displayed on the cards